### ISSUE TITLE
Allow loading custom logos at startup

### ISSUE THREAD

**Issue by carstene1ns** [EasyRPG/Player#623]
Some games have replaced the Enterbrain logo at startup with custom ones.
It is actually a XYZ file embedded in the executable resources and therefore can be extracted.
It would be nice to show a custom logo after the EasyRPG one.
This is analogous to issue #605, `scene_logo` could look for a `logo2.[xyz,png,bmp]` file and load it.

Here are some custom ones:

*El Libro*:
![logo1](http://picload.org/image/paarora/logo.png)

*Ahriman's Prophecy*:
![logo2](http://picload.org/image/paarorl/logo.png)

**Comment by fdelapena** [comment]
As a side note: older RPG_RT versions have 3 logos instead of 1.

**Comment by carstene1ns** [comment]
Yes, but I think it is not really needed to delay the start so much.

**Comment by carstene1ns** [comment]
As a sidenote:
To extract the logos almost any PE resource tool can be used, for example `wrestool` from icoutils.
```bash
wrestool -xR --type='XYZ' -o logo.xyz RPG_RT.exe
```

**Comment by fdelapena** [comment]
Is there any common library available on distros to parse them without cmd calls?

Or copypaste source code from a lightweight implementation, e.g. https://github.com/trailofbits/pe-parse

**Comment by jetrotal** [comment]
Relevant discussion about this subject:

![image](https://github.com/EasyRPG/Player/assets/45118493/80ee3cd8-0021-40db-9944-d9d28efb06d8)

**Comment by jetrotal** [comment]
Implemented a Logo Loader at #3096.

Right now it only read logos from a `Logo` folder near the exe.
I wonder how hard is to adapt the exfont extractor to get the embedded xyz files.

**Comment by Ghabry** [comment]
To skip default logos (as they add no value and delay the startup) I will need a copy of all the standard startup logos in _original_ form (so XYZ, no PNG conversion etc.).

Could somebody do me a favor and help collecting them from RPG_RT.exe?

Then I can hash them and ignore them.

**Comment by jetrotal** [comment]
@Ghabry, some logo I extracted with ResourceHacker:
[Logo.zip](https://github.com/EasyRPG/Player/files/13064615/Logo.zip)

For some reason I failed to get Logo4. It's actually logo1 from an older rm2k3 exe.
![image](https://github.com/EasyRPG/Player/assets/45118493/648fbec1-0baa-4460-8824-ff4cfc4dd687)

**Comment by jetrotal** [comment]
@Ghabry, some logo I extracted with ResourceHacker:
[Logo.zip](https://github.com/EasyRPG/Player/files/13121694/Logo.zip)

![image](https://github.com/EasyRPG/Player/assets/45118493/9df22bb3-6ae7-4e0c-aa89-8672315f8c7e)
organized by 2k, 2k3 and Steam variations.

**Comment by Ghabry** [comment]
CRC32 (If I didn't mess this up xD)

```
2k/LOGO1.xyz 3FD74659
2k/LOGO2.xyz 4C47C4AF
2k/LOGO3.xyz C9E181F7
2k3/LOGO1.xyz 9EB932F4
Steam/LOGO1.xyz 61B31C67
```

**Comment by jetrotal** [comment]
While extracting the the logos I found a problematic exe, where the logo couldn't be extracted.

Ghabry asked for this file to do some extra tests:
[RPG_RT_from_Maker_Universe_BR.zip](https://github.com/EasyRPG/Player/files/13121829/RPG_RT_from_Maker_Universe_BR.zip)

It's from the game "Safira 43 - O Inicio (Melhorado)":

https://www.4shared.com/rar/8L3vzCng/safira-43_o_inicio_melhorado.html
